### PR TITLE
dev - add wallet, cold, hot terms to FAQ

### LINKED ISSUE #344: Uses undefined terms
In 1.6, Create Test Accounts, in section Step 2, the following text appears:

> Notice that actually alice has both owner and active public keys. EOSIO has a unique authorization structure that has added security for your account. You can minimize the exposure of your account by keeping the owner key **cold**, while using the key associated with your active permission. 

This is the first appearance of the term **cold**.  No explanation of that term is provided. So, the reader is left trying to figure out what cold means in this context.

### PR DESCRIPTION
**Pull request by iamveritas**
add wallet, cold, hot terms to FAQ
fixes #344

sister PR: https://github.com/EOSIO/welcome/pull/452

### CODE REVIEW

File: docs/96_faq/index.md [hunk 1]
```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
 
 If you are using a third party wallet with a UI, they may provide the ability to update your account public and private key pairs.
 
+### What is a digital wallet
+
+A digital wallet, also called simply "wallet", is where the keys are stored. A wallet can also allow you to:
```
Comment (bobgt) [inline docs/96_faq/index.md]: Suggestion if it makes sense:

 ".....is where the cryptographic keys are stored."

File: docs/96_faq/index.md [hunk 2]
```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
...
+
+* generate new keys,
```
Comment (bobgt) [inline docs/96_faq/index.md]: For bullets, we want to use the following rules for consistency and Style Guide adherence: 

1. Avoid starting a bullet point with a lowercase letter
2. Avoid using comma or semicolon after a bullet point

So we can change the bullets to:

* Generate new keys
* Delete keys from wallet
* Send, receive, and spend cryptocurrencies

Note: Starting a bullet point with a lower case letter is not incorrect. It's just a different style. For dev docs, we use the above suggested style.

File: docs/96_faq/index.md [hunk 3]
```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
...
+* delete keys from wallet,
+* send, receive, and spend cryptocurrencies,
+
+A digital wallet can be either cold or hot.
+
+### What is a cold wallet
+
+A cold wallet, also called a "hardware wallet", "offline wallet" or "cold storage", is a cryptocurrency wallet stored offline. The cold wallet is stored on a platform that is offline, that is not connected to the internet. This way the wallet is protected from cyber hacks and other vulnerabilities to which an online system is exposed to. It is the most secure way to store cryptocurrencies.
```
Comment (bobgt) [inline docs/96_faq/index.md]: Wanna add "digital wallet"? As in the digital wallet definition, cold and hot is mentioned. So how about:

"A cold wallet, also called a "hardware wallet", "offline wallet" or "cold storage", is a type of digital cryptocurrency wallet stored offline."

File: docs/96_faq/index.md [hunk 4]
```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
...
+
+### What is a hot wallet
+
+A hot wallet is a cryptocurrency wallet stored on a platform that is online, connected to the internet, e.g. web hosted, mobile hosted, or hosted by an exchange. Because of this aspect the hot wallets are considered less secured than the cold wallets albeit they provide more flexibility and availability to the users.
```
Comment (bobgt) [inline docs/96_faq/index.md]: Same as cold wallet:

"A hot wallet is a type of digital cryptocurrency wallet stored on a..."

**Review by bobgt — APPROVED** [review]

File: docs/30_getting-started-guide/20_local-development-environment/50_create-dev-accounts.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ cpu bandwidth:
      available:          unlimited
      limit:              unlimited
 ```
-Notice that actually `alice` has both `owner` and `active` public keys. EOSIO has a unique authorization structure that has added security for your account. You can minimize the exposure of your account by keeping the owner key cold, while using the key associated with your `active` permission. This way, if your `active` key were ever compromised, you could regain control over your account with your `owner` key.
+Notice that actually `alice` has both `owner` and `active` public keys. EOSIO has a unique authorization structure that has added security for your account. You can minimize the exposure of your account by keeping the owner key stored in a [cold wallet](https://developers.eos.io/welcome/latest/faq/index/#what-is-a-cold-wallet), while using the key associated with your `active` permission. This way, if your `active` key were ever compromised, you could regain control over your account with your `owner` key.
 
 In term of authorization, if you have a `owner` permission you can change the private key of `active` permission. But you cannot do so other way around.
```
Comment (lparisc) [inline docs/30_getting-started-guide/20_local-development-environment/50_create-dev-accounts.md]: In term of authorization --> In term**s** of authorization
so other way --> so **the** other way

File: docs/96_faq/index.md [hunk 6]
```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
 
 If you are using a third party wallet with a UI, they may provide the ability to update your account public and private key pairs.
```
Comment (lparisc) [inline docs/96_faq/index.md]: your account public... --> your account's public...

**Review by lparisc — APPROVED** [review]
Few suggestions.

**Pull request merged**